balena/models/device_os.py

```diff
@@ -68,14 +68,21 @@ def get_config(self, app_id, options):
             endpoint=self.settings.get('api_endpoint')
         )
 
-    def download(self, raw=None, **data):
+    def download(self, raw=False, **data):
```

Comment:
Optional-if-you-agree: It would be nice to default to `latest` in here as well, just for the shake of consistency.
If the required methods for getting the app's device type are not on hand, then that's fine for later.
Alternatively we could drop `'latest'` from `download_unconfigured_image` just to make these consistent and avoid user confusion/errors.